builder/comp-builder.nix

```diff
@@ -77,6 +77,16 @@ let self =
 }@drvArgs:
 
 let
+  # Ignore attempts to include DWARF info when it is not possible
+  enableDWARF = drvArgs.enableDWARF or false
```

Comment:
Would it be better to have something like this as an assert? As it is, it'll just not do anything silently, which seems like it might trip people up.